examples/express-graphql-http-functions/server.ts

```diff
@@ -0,0 +1,46 @@
+import { readFileSync } from "fs";
+import { join as pathJoin } from 'path'
+
+import express from "express";
+import { createHandler } from 'graphql-http/lib/use/express';
+
+import { extractGratsSchemaAtRuntime, buildSchemaFromSDL } from "grats";
+import { Context, GroupService, UserService } from "./context";
+
+async function main() {
+  const app = express();
+
+  const schema = getSchema();
+
+  app.post(
+    "/graphql",
+    createHandler<Context>({
+      schema: schema,
+      rootValue: null,
+      context() {
+        return {
+          userService: new UserService(),
+          groupService: new GroupService(),
+        } satisfies Context;
```

Comment:
TIL

Reply:
I'm a big fan of this TS feature; however we don't really need it here, since we already enforcing a type right on the  `createHandler<Context>` call site. I'll remove it.